resources/views/themes/default1/order/show.blade.php

```diff
@@ -20,6 +20,9 @@
   a {
     color: currentColor;!important;
   }
+  .brtags br {
```

Comment:
any reason for making its display as none?

Reply:
Fixed